manuelshub/main.c

```diff
@@ -0,0 +1,70 @@
+#include "calc.h"
+
+int main()
+{
+	double num1, num2, result;
+	char operator;
+
+	sleep(1.5);	
+	printf("\tHi there!!\n");
+	sleep(2.75);
+	printf("\tYou're most Welcome to use our CALCULATOR!!\n");
+	sleep(2.75);
+	printf("\nPERMITTED OPERATORS:\n");
+	sleep(2.5);
+	printf("For Addition (+)\n");
+	sleep(2);
+	printf("For Subtraction (-)\n");
+	sleep(2);
+	printf("For Multiplication (*)\n");
+	sleep(2);
+	printf("For Division (/)\n");
+	sleep(2);
+	printf("\nEnter The first number-->> ");
+	scanf("%2lf", &num1);
+	sleep(1.5);
+	printf("Enter the second number-->> ");
+	scanf("%2lf", &num2);
+	sleep(1.5);
+	printf("Enter the arithmetic sign:");
+	scanf(" %c", &operator);
+	sleep(1.5);
+
+	switch(operator)
+	{
+		case '+':
+			result = add(num1, num2);
+			break;
+
+		case '-':
+			result = sub(num1, num2);
+			break;
+
+		case '*':
+			result = mul(num1, num2);
+			break;
+
+		case '/':
+			result = div(num1, num2);
+			break;
+
+		default:
+			printf("Invalid Operator!!\n");
+			return (1);
+	}
+
+	if (result == 0.00 /*|| result < 0.00 || result > 0.00*/)
```

Comment:
Again, what happens when the user performs any of the following:

* Sum of -2 and 2
* Difference between 2 and 2

Won't that produce 0, therefore, making this condition true, ergo treating that as an invalid operation?